docs/book/src/component-config-tutorial/testdata/project/config/crd/kustomization.yaml

```
@@ -16,6 +16,8 @@ patches:
 #- path: patches/cainjection_in_projectconfigs.yaml
 #+kubebuilder:scaffold:crdkustomizecainjectionpatch
 
+# [WEBHOOK] To enable webhook, uncomment the following section
 # the following config is for teaching kustomize how to do kustomization for CRDs.
-configurations:
-- kustomizeconfig.yaml
+
+# configurations:
+# - kustomizeconfig.yaml
```

```suggestion
#configurations:
#- kustomizeconfig.yaml
```
We cannot have the space 
Also, we should automatically uncomment this one when a webhook is scaffold.